MiaWrapper.py

```diff
@@ -37,7 +47,7 @@ def sortImages(images, modalities):
                 currentImageLocation = modalityImage['location']
                 filename = os.path.basename(currentImageLocation)
                 newImageLocation = os.path.join(newImageDirectory, filename)
-                os.rename(currentImageLocation, newImageLocation)
+                copyfile(currentImageLocation, newImageLocation)
```

Comment:
Perhaps a simpler alternative to the checkIfCopyIsNeeded solution:

if not os.path.isfile(newImageLocation)
   copyFile(currentImageLocation, newImageLocation)

This will check all files and is much less code.